src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -3,7 +3,7 @@ book_path: /web/updates/_book.yaml
 description: The new Native File System API enables developers to build powerful web apps that interact with files on the user's local device, like IDEs, photo and video editors, text editors, and more. After a user grants a web app access, this API allows web apps to read or save changes directly to files and folders on the user's device.
```

Comment:
'Powerful' is a severely overused term in DevRel articles. I don't have anything to suggest so please keep your ears open for something fresher.

Reply:
Will do!